gdcdictionary/schemas/program.yaml

```diff
@@ -36,6 +36,7 @@ properties:
   name:
     type: string
     description: "Full name/title of the program."
+    pattern: "^[A-Z0-9]+|[0-9]+|[A-Z]+$"
```

Comment:
is this saying the name should start with uppercase alphabetical or number and end with only uppercase alphabetical?